src/debug/debug.cc

```diff
@@ -1219,7 +1220,7 @@ _db_rotate_log(void)
 }
 
 static const char *
-debugLogTime(const time_t t)
+debugLogTime(const timeval &t)
 {
     struct tm *tm;
```

Comment:
If we are modifying all setters, then let's remove this line (and use `const auto` or `auto` in setters).